pkg/logs/epforwarder/epforwarder.go

```diff
@@ -0,0 +1,158 @@
+package epforwarder
```

Comment:
Why is this in `pkg/log` and not `pkg/forwarder` or simply `pkg` ?

Reply:
Since this is mostly relying on `pkg/logs` code I think it should be there instead of `pkg/forwarder`. 

Note the set of imports:
```
	"github.com/DataDog/datadog-agent/pkg/logs/auditor"
	"github.com/DataDog/datadog-agent/pkg/logs/client"
	"github.com/DataDog/datadog-agent/pkg/logs/client/http"
	"github.com/DataDog/datadog-agent/pkg/logs/config"
	"github.com/DataDog/datadog-agent/pkg/logs/message"
	"github.com/DataDog/datadog-agent/pkg/logs/restart"
	"github.com/DataDog/datadog-agent/pkg/logs/sender"
	"github.com/DataDog/datadog-agent/pkg/util/log"
```

Where else could we put it? Do you think it should be a top-level package `pkg/epforwarder`?

Reply:
I think so yes. It's not related to logs even if it use some internal of the `logs` package. I think ` pkg/epforwarder` makes more sens.

Reply:
Ok, that works. Moved https://github.com/DataDog/datadog-agent/pull/7735/commits/1962127745199e0a07b354bab93d1da2903cf588